packages/ursula/src/front.js

```diff
@@ -21,52 +21,71 @@ export default function (part) {
   store.set('xScale', utils.stretchToScale(options.fabricStretch))
 
   // Design pattern here
+  
+  // determine height of front part: use cross seam (and cross seam front) if selected and available
+  if (options.useCrossSeam && measurements.crossSeam) {
+	store.set('crossSeam',measurements.crossSeam)
```

Comment:
It looks like there are some tabs in here. Would you mind replacing them with 2 spaces?
```suggestion
    store.set('crossSeam',measurements.crossSeam)
```

Reply:
Done!